Gemfile

```diff
@@ -9,6 +9,7 @@ gem "bootstrap-sass", "3.4.1"
 gem "bootstrap-will_paginate", "1.0.0"
 gem "config"
 gem "faker", "2.1.2"
+gem "figaro"
```

Comment:
gem này để làm gì e?

Reply:
Sử dụng gem Figaro có thể giúp bạn chuẩn hoá toàn bộ các dự án tham gia một cách dễ dàng

Reply:
Sử dụng gem Figaro có thể giúp chuẩn hoá toàn bộ các dự án tham gia một cách dễ dàng

Reply:
k đúng, e tìm hiểu thêm nhé

Reply:
> k đúng, e tìm hiểu thêm nhé

e trả lời lại này

Reply:
Figaro được viết để giúp dễ dàng cấu hình các ứng dụng Rails một cách an toàn.
Giá trị cấu hình thường bao gồm thông tin nhạy cảm. Figaro cố gắng bảo mật theo mặc định bằng cách khuyến khích một quy ước giữ cấu hình ngoài Git.